src/libs/antares/study/scenario-builder/TSnumberData.cpp

```diff
@@ -488,6 +517,97 @@ bool renewableTSNumberData::reset(const Study& study)
     return true;
 }
 
+
+// ================================
+// Transmission capacities ...
+// ================================
+bool ntcTSNumberData::reset(const Study& study)
+{
+    assert(&study != nullptr);
```

Comment:
This assert is probably useless

Reply:
ok, you probably refer to the only call of **ntcTSNumberData::reset()** made in **Rules::reset()**, where this assert is already called. But what if reset is called somewhere else someday ? We  could forget to call this assert then.
For me, this assert should be called in **ntcTSNumberData::reset()**.
Note that this assert is also called in **ntcTSNumberData** siblings (I mean her sisters, like **thermalTSNumberData**)